conf.yaml

```diff
@@ -120,9 +120,10 @@ contents:
         sections:
           - title:      "Welcome to Elastic"
             prefix:     en/welcome-to-elastic
-            current:    main
+            current:    *stackcurrent
             index:      welcome-to-elastic/index.asciidoc
-            branches:   [ {main: master} ]
+            branches:   [ 8.1 ]
+            live:       *stacklivemain
```

Comment:
I'm not sure if this will work, since `stacklivemain` includes branches (7.17) that don't exist for this book. You can leave `live` out entirely and it will default to the full list of `branches`, which I think is fine for now.

Reply:
We can make a 7.17 branch if needed?

Reply:
I made one. :)